### PR TITLE
[WIP] CLM - Filter UI Improvements and Refactor

### PR DESCRIPTION
**Pull request by lneves12**
## What does this PR change?

**add description**

## GUI diff

No difference.

Before:

After:

- [ ] **DONE**

## Documentation
- No documentation needed: **add explanation. This can't be used if there is a GUI diff**
- [doc-susemanager](https://github.com/SUSE/doc-susemanager) PR or issue was created (GitHub automatic link expected below)

- [ ] **DONE**

## Test coverage
- No tests: **add explanation**
- Unit tests were added
- Cucumber tests were added

- [ ] **DONE**

## Links

Fixes #
Tracks # **add downstream PR, if any**

- [ ] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: web/html/src/manager/content-management/shared/business/filters.enum.js [hunk 1]
```diff
@@ -1,20 +1,133 @@
...
+    longDescription: ' containing '
+  },
+  CONTAINS_PKG_NAME: {
+    key: 'contains_pkg_name',
+    text: 'contains (><)',
+    longDescription: ' contains package name '
+  },
+  CONTAINS_PKG_LT_EVR: {
+    key: 'contains_pkg_lt_evr',
+    text: 'version lower than',
```
Comment (mcalmer) [inline web/html/src/manager/content-management/shared/business/filters.enum.js]:
```suggestion
    text: 'version lower than (<)',
```

File: web/html/src/manager/content-management/shared/business/filters.enum.js [hunk 2]
```diff
@@ -1,20 +1,133 @@
...
+    longDescription: ' contains package with version lower than '
+  },
+  CONTAINS_PKG_LE_EVR: {
+    key: 'contains_pkg_le_evr',
+    text: 'version lower or equal than',
```
Comment (mcalmer) [inline web/html/src/manager/content-management/shared/business/filters.enum.js]:
```suggestion
    text: 'version lower or equal than (<=)',
```

File: web/html/src/manager/content-management/shared/business/filters.enum.js [hunk 3]
```diff
@@ -1,20 +1,133 @@
...
+    longDescription: ' contains package with version lower or equal than '
+  },
+  CONTAINS_PKG_EQ_EVR: {
+    key: 'contains_pkg_eq_evr',
+    text: 'version equal',
```
Comment (mcalmer) [inline web/html/src/manager/content-management/shared/business/filters.enum.js]:
```suggestion
    text: 'version equal (=)',
```

File: web/html/src/manager/content-management/shared/business/filters.enum.js [hunk 4]
```diff
@@ -1,20 +1,133 @@
...
+    longDescription: ' contains package with version equal than '
+  },
+  CONTAINS_PKG_GE_EVR: {
+    key: 'contains_pkg_ge_evr',
+    text: 'version greater or equal than',
```
Comment (mcalmer) [inline web/html/src/manager/content-management/shared/business/filters.enum.js]:
```suggestion
    text: 'version greater or equal than (>=)',
```

File: web/html/src/manager/content-management/shared/business/filters.enum.js [hunk 5]
```diff
@@ -1,20 +1,133 @@
...
+    longDescription: ' contains package with version greater or equal than '
+  },
+  CONTAINS_PKG_GT_EVR: {
+    key: 'contains_pkg_gt_evr',
+    text: 'version greater than',
```
Comment (mcalmer) [inline web/html/src/manager/content-management/shared/business/filters.enum.js]:
```suggestion
    text: 'version greater than (>)',
```

**Comment by lneves12** [comment]
wrong name branch: closed to

**Pull request closed without merging**